vault_program/src/initialize_vault_update_state_tracker.rs

```diff
@@ -75,12 +75,28 @@ pub fn process_initialize_vault_update_state_tracker(
         &vault_update_state_tracker_seeds,
     )?;
 
-    let additional_assets_need_unstaking = vault
-        .calculate_additional_supported_assets_needed_to_unstake(
+    let additional_assets_need_unstaking = if vault.additional_assets_need_unstaking() > 0 {
+        // indicates an old partially cranked state
+        // no vault operations can be done until a full update cycle has run
+        // this means that the `additional_assets_need_unstaking` will not change
+        // between epochs and we can effectively carry on where we left off
```

Comment:
worth acknowledging tradeoffs here for vault not being able to update withdrawal fee or program fee not taking effect

Reply:
My thoughts. program fee will not change much, and it's the vault's fault for not updating fully